### ISSUE TITLE
EIA subtitles support

### ISSUE THREAD

**Issue by Nadahar** [DigitalMediaServer/DigitalMediaServer#45]
@Sami32 I remember that you've talked about this, but I can't remember what was the issue. I now have some files with EIA-608 subtitles and would like to get them to work. Can you remind me what you already know about the issue?

**Comment by Sami32** [comment]
AFAICR it was working, you gived me a hand on the MediaInfo fix for Teletext.
I was only waiting for the end of this year where EIA-708 support is planned.

**Comment by Nadahar** [comment]
Ok, you mean support in FFmpeg? We could/should still parse it correctly and then it would be little job to implement when FFmpeg gets support.

**Comment by Sami32** [comment]
Yes, FFmpeg support.
Sure.

Where did i said that?

**Comment by Sami32** [comment]
One was just the parsing and the other FFmpeg implementation if i remember correctly.

**Comment by Nadahar** [comment]
The commits I linked are part of https://github.com/Sami32/UniversalMediaServer/pull/8, so those can't be what you meant.

**Comment by Sami32** [comment]
Woua that is very long time, i cannot remember now :(

**Comment by Sami32** [comment]
I can remember having done others subtitles enhancements, like for scaling, image subtitle support, video stream correct detection enhancement.
But that could be done later, if i ever remember where.
ok, now i remember having stopped Java on the final step to fix image subtittle as cloning get my well know patience out ;P

**Comment by Nadahar** [comment]
Yes, I think it's fine to separate stuff up to make it easier to review and test, but when you say "depend on" I understand it like there are some other coded that is needed for this to work.

**Comment by Sami32** [comment]
Yes, i should have said that for a reason that i don't remember now.

**Comment by Nadahar** [comment]
> Yes, i should have said that for a reason that i don't remember now.

That's why it's smart to write exactly what it depends on :stuck_out_tongue_closed_eyes:
